src/styles/components/_component-card.scss

```diff
@@ -6,14 +6,14 @@
 
   .component-card {
     margin: 0;
-    border: 2px solid var(--foreground-color);
+    border: 2px solid var(--background-color);
```

Comment:
I think we want `var(--foreground-color)` here, then `var(--background-color)` in the media query down below for dark mode.

Reply:
Fixed!